src/dsmlp/app/validator.py

```diff
@@ -120,17 +126,28 @@ def validate_pod(self, request: Request):
 
 #        if 'k8s-sync' in namespace.labels:
         user = self.awsed.describe_user(username)
+        if not user:
+            raise ValidationFailure(f"namespace: no AWSEd user found with username {username}")
         allowed_uid = user.uid
+        allowed_courses = user.enrollments
```

Comment:
comment somewhere that user.enrollments is a list of strings. Even though this is said in the clients dataclass, this is a little against how we normally use enrollments and so specification here would be good